app/controllers/spree/admin/products_controller.rb

```diff
@@ -31,6 +31,19 @@ class ProductsController < ResourceController
         }
       } }
 
+      respond_override update: { html: {
+        success: lambda {
+          redirect_to edit_admin_product_url(@product, @url_filter)
+        },
+        failure: lambda {
+          redirect_to edit_admin_product_url(@product, @url_filter)
+        }
+      } }
```

Comment:
We need to stop using this respond_override style.
Can you please implement this in the update action? you can bring all the code from resource_controller, it would start with something like this and then you can add these overrides in there:
![image](https://user-images.githubusercontent.com/1640378/86457524-b532e680-bd1b-11ea-9f5f-c14883486658.png)


I wonder if the update action really needs to support html and js, maybe not and one can be removed.

Reply:
Sure, I believe the js shouldn't be needed.

Reply:
Done, I also refactored the create method to remove "respond_override".